include/bluetooth/hci.h

```diff
@@ -204,6 +204,10 @@ struct bt_hci_cmd_hdr {
 						BT_LE_FEAT_BIT_SLAVE_FEAT_REQ)
 #define BT_FEAT_LE_DLE(feat)                    BT_FEAT_TEST(feat, 0, 0, \
 						BT_LE_FEAT_BIT_DLE)
+#define BT_FEAT_LE_PHY_2M(feat)                 BT_FEAT_TEST(feat, 0, 1, \
+						BT_LE_FEAT_BIT_PHY_2M - 8)
+#define BT_FEAT_LE_PHY_COD(feat)                BT_FEAT_TEST(feat, 0, 1, \
+						BT_LE_FEAT_BIT_PHY_COD - 8)
```

Comment:
This looks weird. The BT_FEAT_TEST() macro takes as input an octet offset and the bit number within that octet, so passing it a continous enumeration that disregards the per-octet access doesn't seem right. Looking at the core spec it seems the way LE feature bits are defined is different from non-LE features. The former is a continuous enumeration whereas the latter specifies the bits as octests and bits within the octet. From this I'd conclude that the current BT_FEAT_TEST() macro is not as-is suitable for being used for LE features. I see two options:

Add a new BT_LE_FEAT_TEST() macro that only takes the features pointer and the bit number as input (no octet) and infers the octet from the bit number. Alternatively, make the existing BT_FEAT_TEST perform modulo 8 on its last parameter so you don't have to do the "- 8" magic. I think a new macro might be the cleaner way to proceed.